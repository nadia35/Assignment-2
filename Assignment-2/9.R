reverseLastThree <- function(values){
  rev(values)
  
  if(length(values) > 3){
    values <- values[(length(values) - 2) : length(values)]
    return(rev(values))
  }
  else{
    return(rev(values))
  }
  
  
}
