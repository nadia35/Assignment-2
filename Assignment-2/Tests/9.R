source("9.R")

print(reverseLastThree(c(0, 1, 2, 3, 4, 5, 6, 7, 8, 9)))
print(reverseLastThree(c(9, 8, 7, 6, 5, 4, 3, 2, 1, 0)))
print(reverseLastThree(c(0, 0, 0, 0, 0, 0, 0, 1)))
print(reverseLastThree(c(98.19, 800.01, 561.24)))
print(reverseLastThree(c(-2.1, 0.3)))