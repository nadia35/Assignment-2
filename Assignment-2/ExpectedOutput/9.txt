[1] 9 8 7
[1] 0 1 2
[1] 1 0 0
[1] 561.24 800.01  98.19
[1]  0.3 -2.1
